WordPress/src/test/java/org/wordpress/android/ui/JetpackConnectionSourceTest.kt

```diff
@@ -0,0 +1,42 @@
+package org.wordpress.android.ui
+
+import org.assertj.core.api.Assertions
+import org.junit.Test
+import org.wordpress.android.BaseUnitTest
+
+class JetpackConnectionSourceTest : BaseUnitTest() {
```

Comment:
1) Praise (❤️): Kudos for adding this test suite!
2) Suggestion (💡): For next time, consider adding the test before making the refactor, since this will give you much more confidence that the refactor succeeded.
3) Minor (🔍): Consider moving the `Assertions` as an import.
4) Minor (🔍): Consider splitting the `given/when/then` logic with an empty line (the `value/result/assertThat` lines).